packages/riverpod_lint_flutter_test/test/goldens/lints/avoid_use_ref_inside_dispose.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/widgets.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+
+final provider = Provider((ref) => 0);
+
+class MyWidget extends ConsumerStatefulWidget {
```

Comment:
The following should not trigger a warning:

```suggestion
class PlainClass with ChangeNotifier {
  PlainClass(this.ref);
  final WidgetRef ref;
  @override
   void dispose() {
     ref.read(provider);
   }
}
```